Game/KinkyDungeonRestraints.js

```diff
@@ -3087,20 +3124,13 @@ function KDGetRestraintsEligible(enemy, Level, Index, Bypass, Lock, RequireWill,
 	let RestraintsList2 = [];
 	if (!options?.allowLowPower) {
 		// Only does strictest for that group
-		let noway = KinkyDungeonStatsChoice.get("NoWayOut");
```

Comment:
Well this was probably a bug.

First of all, everything here happens within the `allowLowPower == false` block, so I suppose we want to choose the strongest one all the time.

The behavior before the change:
1. Find if any group has anything worn
2. If the group doesn't have anything to worn, OR "NoWayOut" is set, OR (this is the more problematic part) no restraint was chosen for the group, go to 3, otherwise exit;
3. If any restraint down the list is not weaker than the chosen one, either replace the chosen one with the new and stronger one, or add the equally strong one to the list, go back to 2.

The problem is after 3 is run once there will always be something chosen, so unless "NoWayOut" is on, 2 will always be false after one round. Therefore, only the restraint on the top of the list (not necessarily the strongest) gets chosen if "NoWayOut" is off, I suppose this is not the original intention (at least all restraints equally strong should have a fair chance being chosen, otherwise those later in the list will be very sad XD)?

After the change: regardless of the "NoWayOut" choice, there will always be a strongest restraint, or a group of equally strongest restraints chosen in the final list.